### PR TITLE
Update plan-for-teams-live-events.md

### PR DESCRIPTION
**Pull request by sukihanabi**
#1830 
Actions Taken
	- added link "Get started with MS Teams"article: https://support.office.com/en-us/article/get-started-with-microsoft-teams-live-events-d077fec2-a058-483e-9ab5-1494afda578a

	- Merged, Powershell command for Teams Policy and update the prerequisite on "Who can schedule a live meeting"
Ref: "Get started with MS Teams"article: https://support.office.com/en-us/article/get-started-with-microsoft-teams-live-events-d077fec2-a058-483e-9ab5-1494afda578a
add ref: https://support.office.com/en-us/article/get-started-with-microsoft-teams-live-events-d077fec2-a058-483e-9ab5-1494afda578a

	- For Skype for Business use Skype Meeting Broadcast while in TeamsOnly kindly use Teams Live Event.
	REF: What are Microsoft Teams live events? https://docs.microsoft.com/en-us/microsoftteams/teams-live-events/what-are-teams-live-events

	- Teams live events are considered the next version of Skype Meeting Broadcast
	Ref: Live events in Microsoft Teams https://techcommunity.microsoft.com/t5/Microsoft-Teams-Blog/Live-events-in-Microsoft-Teams/ba-p/214955
	- Added Teams Live Event with Coexistence of Teams and Skype for Business
https://docs.microsoft.com/en-us/microsoftteams/teams-and-skypeforbusiness-coexistence-and-interoperability

### CODE REVIEW

File: Teams/teams-live-events/plan-for-teams-live-events.md [hunk 1]
```diff
@@ -24,22 +24,32 @@ When you are planning Teams live events to hold large meetings in your organizat
...
+
+> Teams live events are considered the next version of Skype Meeting Broadcast and will eventually replace the capabilities provided in Skype Meeting Broadcast. At this point, Microsoft will continue to support Skype Meeting Broadcast for users who are using Skype for Business in their organizations, with no disruption in service for new or future events. However, we encourage you to try out Teams live events to leverage all the new and exciting features including screen sharing and support for external hardware/software encoders.
 It's important to know that an Office 365 license is required to participate in a live event as an authenticated user but this depends on the production method used:
 
 - **For Quick start production**  The user must be assigned a Microsoft Teams license.
 - **For External encoder production** The user must be assigned a Microsoft Stream license.
 
-For more information on licensing, see [Microsoft Teams add-on licensing](../teams-add-on-licensing/microsoft-teams-add-on-licensing.md).
+For more information, see [Microsoft Teams live events production method](https://docs.microsoft.com/en-us/microsoftteams/teams-live-events/what-are-teams-live-events)
```
Comment (nenonix) [inline Teams/teams-live-events/plan-for-teams-live-events.md]: We have to remove en-us part of all URLS 
https://docs.microsoft.com/microsoftteams/teams-live-events/what-are-teams-live-events
but I would suggest you  use the link to the production section like this:
**For more information, see [Microsoft Teams live events production] (https://docs.microsoft.com//microsoftteams/teams-live-events/what-are-teams-live-events#production).**

Comment (sukihanabi) [reply Teams/teams-live-events/plan-for-teams-live-events.md]: Duly noted and thanks! @nenonix  & @LolaJacobsen  , removed all URLS that has en-us .

Comment (LolaJacobsen) [reply Teams/teams-live-events/plan-for-teams-live-events.md]: @nenonix - for links within the same docset, you should use relative links. For example:
[Microsoft Teams live events production](../teams-live-events/what-are-teams-live-events#production).

**Comment by LolaJacobsen** [comment]
@tonysmit - you're listed as MS.Reviewer for this one. Please review the big added section, in this view:
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/1981/commits/6ffe47c6da9dfbff9c77151528f472a39eec1282

**Comment by LolaJacobsen** [comment]
@bjwhalen please review the "Teams Live Event with Coexistence of Teams and Skype for Business" section, in this view of the PR: https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/1981/commits/6ffe47c6da9dfbff9c77151528f472a39eec1282

**Comment by LolaJacobsen** [comment]
Note to @tonysmit and @bjwhalen - I cleaned up URLs in later commits. I'll make sure all the URLs are properly crafted before I merge this PR.

**Comment by scanum** [comment]
@LolaJacobsen Hi Lola, could you please review changes made by @sukihanabi? Thanks!

**Pull request merged**